lib/core/src/traits.rs

```diff
@@ -0,0 +1,8 @@
+use nalgebra::Vector3;
```

Comment:
The name of this file is very bad. It does not tell semantically what is here. I would never think that it contains `Position` getter and setter. Moreover, this file will become a big garbage soon if we allow for it becuase everyone will put there things.

Reply:
@wdanilo I agree. Any suggestion where I should move `HasPosition` to?

Reply:
@notdanilo as agreed, leave it somewhere and Ill fix its placement when refactoring files